general/releases/4.5.md

```diff
@@ -42,6 +43,10 @@ Since Moodle 4.3, the maximum length for the database prefix (`$CFG->prefix`) is
 
 :::
 
+#### Aurora MySQL compatibility version
+
+This system requires configuration using standard MySQL version numbers (e.g., 8.0) rather than Aurora's internal versioning (e.g., 3.x) to maintain compatibility with MySQL client tools, version-dependent features, and database detection methods.
```

Comment:
We try to avoid use of `e.g.` and `i.e.` -- use `for example` and `that is`.

```suggestion
This system requires configuration using standard MySQL version numbers (e.g., 8.0) rather than Aurora's internal versioning (for example, 3.x) to maintain compatibility with MySQL client tools, version-dependent features, and database detection methods.
```